convert ERROR

![image](https://github.com/user-attachments/assets/ba4c2a2a-4764-4b18-885b-0f6c1c760138)

> When I load a model using OpenCV, I get the following error
> 
> OpenCV(4.5.4) F:\repos\opencv-4.5.4\modules\dnn\src\onnx\onnx_importer.cpp:739: error: (-2:Unspecified error) in function 'cv::dnn::dnn4_v20211004::ONNXImporter::handleNode'
> 
> > Node [Conv]:(Convolution_21635_output2) parse error: OpenCV(4.5.4) F:\repos\opencv-4.5.4\modules\dnn\src\layers\layers_common.cpp:106: error: (-5:Bad argument) kernel_size (or kernel_h and kernel_w) not specified in function 'cv::dnn::util::getKernelSize'

For OpenCV, it seems it needs `kernel_shape` attribute while it is not a mandatory attribute as onnx describes [here](https://onnx.ai/onnx/operators/onnx__Conv.html).

> kernel_shape - INTS :
> The shape of the convolution kernel. If not present, should be inferred from input W.

> > When I load a model using OpenCV, I get the following error
> > OpenCV(4.5.4) F:\repos\opencv-4.5.4\modules\dnn\src\onnx\onnx_importer.cpp:739: error: (-2:Unspecified error) in function 'cv::dnn::dnn4_v20211004::ONNXImporter::handleNode'
> > > Node [Conv]:(Convolution_21635_output2) parse error: OpenCV(4.5.4) F:\repos\opencv-4.5.4\modules\dnn\src\layers\layers_common.cpp:106: error: (-5:Bad argument) kernel_size (or kernel_h and kernel_w) not specified in function 'cv::dnn::util::getKernelSize'
> 
> For OpenCV, it seems it needs `kernel_shape` attribute while it is not a mandatory attribute as onnx describes [here](https://onnx.ai/onnx/operators/onnx__Conv.html).
> 
> > kernel_shape - INTS :
> > The shape of the convolution kernel. If not present, should be inferred from input W.

![image](https://github.com/user-attachments/assets/213c0a42-4e3c-4155-bf05-e486855beb0a)
I converted the model, but I still get the same error

--- a/openvino2onnx/domain/intel/openvino/ir/ir11.py
+++ b/openvino2onnx/domain/intel/openvino/ir/ir11.py
@@ -78,7 +78,7 @@ def _load_const(graph, model_bin):
             if const_node["shape"]:
                 shape = list(map(int, const_node["shape"].split(",")))
             else:
-                shape = [-1]
+                shape = []
             if np.prod(shape) == 0:
                 data = np.empty([], dtype=dtype)
             else:

--- a/openvino2onnx/domain/intel/openvino/passes/op/normalize_l2.py
+++ b/openvino2onnx/domain/intel/openvino/passes/op/normalize_l2.py
@@ -22,8 +22,8 @@ class NormalizeL2(BaseNodeConversion):
 
     def replace(self, graph: OnnxGraph, ori_node: NodeProto) -> NodeProto:
         axes = self.get_value(ori_node.input[1])
-        if axes is not None and len(axes) == 1:
-            axis = int(axes[0])
+        if axes is not None and axes.size == 1:
+            axis = int(axes.squeeze())
             return make_node(
                 "LpNormalization",
                 inputs=[ori_node.input[0]],

